docs/reference/action_list.rst

```diff
@@ -211,6 +211,115 @@ to more field types, see `SonataDoctrineORMAdminBundle Documentation`_.
     values so use the ``inverse`` option if you really cannot find a good enough
     antonym for the name you have.
 
+Symfony Data Transformers
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If the entity field has a limited list of values (enumeration), it is convenient to use a value object to control
```

Comment:
If this feature is compatible with other model managers than ORM, we should use the term "model" instead of "entity".